website/versioned_docs/version-v11.0.0/guided-tour/refetching/refetching-fragments-with-different-data.md

```diff
@@ -70,7 +70,7 @@ function CommentBody(props: Props) {
 Let's distill what's happening in this example:
 
 * `useRefetchableFragment` behaves similarly to [`useFragment`](../../../api-reference/use-fragment/) (see the [Fragments](../../rendering/fragments/) section), but with a few additions:
-    * It expects a fragment that is annotated with the `@refetchable` directive. Note that `@refetchable` directive can only be added to fragments that are "refetchable", that is, on fragments that are on `Viewer`, on `Query`, on any type that implements `Node` (i.e. a type that has an `id` field), or on a [`@fetchable`](https://fb.workplace.com/groups/graphql.fyi/permalink/1539541276187011/) type.
+    * It expects a fragment that is annotated with the `@refetchable` directive. Note that `@refetchable` directive can only be added to fragments that are "refetchable", that is, on fragments that are on `Viewer`, on `Query`, on any type that implements `Node` (i.e. a type that has an `id` field), or on a `@fetchable` type.
```

Comment:
There is also an article i wrote about it — which we can also migrate into the docs perhaps?

https://dev.to/marais/fetchable-with-relay-1meb